frontend/src/views/critical-ratio.js

```diff
@@ -0,0 +1,112 @@
+/** @jsx jsx */
+import React from "react";
+import { jsx } from "@emotion/core";
+import plane from "../images/plane.png";
+import { useRound } from "../selectors";
+import { useAddCriticalRatio } from "../action-hooks";
+import { Button } from "../components/ui";
+
+function CriticalRatio() {
+  const [criticalRatio, setCriticalRatio] = React.useState("");
+  const [isEnabled, setIsEnabled] = React.useState(false);
+
+  const addCriticalRatio = useAddCriticalRatio();
+
+  const state = useRound();
+
+  const buttoncomponent = {
+    width: 170,
+    display: "block",
+    backgroundColor: "black",
+    color: "white",
+    padding: "14px 20px",
+    margin: "10px auto",
+    border: "none",
+    borderRadius: 4,
+    cursor: "pointer",
+    fontSize: "15px",
+    letterSpacing: ".25px",
+    justifyContent: "center"
+  };
+
+  const styleInput = {
+    margin: "10px auto",
+    background: "none",
+    border: "1px solid black",
+    borderRadius: ".25rem",
+    boxSizing: "border-box",
+    display: "block",
+    fontSize: "1rem",
+    padding: ".5rem",
+    color: "#34495e",
+    width: "100%",
+    "&:focus": {
+      outline: "none",
+      borderColor: "rgba(242, 107, 117, 0.5)"
+    }
+  };
+
+  const contenedor = {
```

Comment:
What do you think about using only English for variables' names?

Reply:
I forgot to delete it too, yes I will change it